preprocessing/text/numbers.py

```diff
@@ -0,0 +1,47 @@
+import re
+
+from num2words import num2words
+
+# TODO: add first second third and cardinals
+class Numbers:
```

Comment:
In my opinion it is heavily overengineered to have a general expansion like this. Are you sure it covers all languages?

Reply:
The num2words library looks quite extensive. The problem though is that it doesn't have a lot of stars.